### PR TITLE
Add ABP.IO / web-framework

### PR DESCRIPTION
**Pull request by m-aliozkaya**
ABP framework is the successor of the open source [ASP.NET Boilerplate](https://aspnetboilerplate.com/) framework. ABP framework is also open source and free. 

See https://github.com/abpframework/abp

### Website
https://abp.io/

### Examples
https://abp.io/
https://www.eshoponabp.com/
https://www.openeventhub.com/

### CODE REVIEW

File: src/technologies/a.json [hunk 1]
```diff
@@ -58,6 +58,20 @@
     "saas": true,
     "website": "https://commercesuite.aboutyou.com"
   },
+  "ABP.IO": {
+    "cats": [
+      18
+    ],
+    "description": "ABP Framework is a complete infrastructure to create modern web applications by following the best practices and conventions of software development.",
+    "icon": "abp.png",
```
Comment (nurbek91) [inline src/technologies/a.json:66]: 32x32 svg
ABP IO

File: src/technologies/a.json [hunk 2]
```diff
@@ -58,6 +58,20 @@
     "saas": true,
     "website": "https://commercesuite.aboutyou.com"
   },
+  "ABP.IO": {
```
Comment (nurbek91) [inline src/technologies/a.json:61]: ABP Framework instead ABP.IO maybe ?

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate thanks for PR!!
Need some cosmetic changes

File: src/technologies/a.json [hunk 3]
```diff
@@ -216,8 +230,8 @@
     "icon": "aspnetboilerplate.png",
     "implies": "Microsoft ASP.NET",
     "js": {
-      "abp.localization": "",
```
Comment (nurbek91) [inline src/technologies/a.json:219]: Are you sure about the fingerprints? Since I don't know all the details

Comment (m-aliozkaya) [reply src/technologies/a.json:219]: Yes, I tested it in my local version.

File: src/technologies/a.json (hunk 2 above)
Comment (m-aliozkaya) [reply src/technologies/a.json:61]: Thank you

**Review by nurbek91 — COMMENTED** [review]
Thank you, I can't say anything about the fingerprints, I'll trust your knowledge

**Comment by m-aliozkaya** [comment]
Thank you very much 🙏

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**